### PR TITLE
Merge staging into master for ROCm 5.5

### PR DESCRIPTION
**Pull request by daineAMD**

### CODE REVIEW

**Review by TorreZuk — APPROVED** [review]
LGTM

**Review by jzuniga-amd — APPROVED** [review]

**Comment by daineAMD** [comment]
Will merge at 6PM MST unless I hear otherwise. Will wait for develop merge to pass CI after rocSOLVER changes develop version.

**Pull request merged**
